source/_components/airly.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: "Airly"
+description: "Instructions on how to integrate Airly within Home Assistant."
+logo: airly.png
+ha_category:
+  - Health
+ha_release: 0.99
+ha_iot_class: Cloud Polling
+redirect_from:
+ - /components/sensor.airly/
+---
+
+The `Airly` integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location. To generate Airly API key go to [Airly for developers](https://developer.airly.eu/register) page.
+
+## Configuration
+
+To add Airly to your installation, go to Configuration >> Integrations in the UI and enable the Airly integration. By default the values will be taken from the Home Assistant configuration.
+
```

Comment:
Missing an example with required config variables

Reply:
This integration doesn't allow to configuration via `configuration.yaml` so there is no example.

Reply:
In that case we can skip my comment 😄